src/App.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import OrderPage from 'views/OrderPage';
 
 function App() {
-  return <div></div>;
+  return (
+    <div>
```

Comment:
I don't think you need the div if you are returning just the one page component.